ozaria/site/components/cinematic/common/CinematicCanvas.vue

```diff
@@ -1,9 +1,15 @@
 <template>
   <!-- TODO: Canvas needs to be responsive to scaling up and down. -->
   <!-- Currently fixed size to the aspect ratio of our play view. -->
-  <div>
-    <div height="514px" id="cinematic-div" ref="cinematic-div" v-on:click="skipShot">
-      <canvas width="800" height="514" id="cinematic-canvas" ref="cinematic-canvas"></canvas>
+  <div id="cinematic-canvas-div">
+    <div height="850px" id="cinematic-div" ref="cinematic-div" v-on:click="skipShot">
```

Comment:
Can this height also come from the data `height` property?

Reply:
Good catch! This is a bug. The height is set in the css.